src/plugins/ThumbnailGenerator/index.test.js

```diff
@@ -0,0 +1,333 @@
+import ThumbnailGeneratorPlugin from './index'
+import Plugin from '../../core/Plugin'
+
+const delay = duration => new Promise(resolve => setTimeout(resolve, duration))
+
+describe('uploader/ThumbnailGeneratorPlugin', () => {
+  it('should initialise successfully', () => {
+    const redux = new ThumbnailGeneratorPlugin(null, {})
+    expect(redux instanceof Plugin).toEqual(true)
```

Comment:
minor copy-paste nit: description says `uploader/ThumbnailGeneratorPlugin` and this var is named `redux`

Reply:
Ha, alway make at least one silly mistake. Fixed!